packages/css/src/global/global.scss

```diff
@@ -25,6 +25,7 @@
 @import "../../../sources/src/styling/global/select";
 @import "../../../sources/src/styling/global/strong";
 @import "../../../sources/src/styling/global/sub";
+@import "../../../sources/src/styling/global/subtitle";
```

Comment:
Vergelijk alle andere filenames eens met jouw nieuw-aangemaakte file: Wat valt je op?

Reply:
> Maar nu twijfel ik of we deze stijling dan in de `css` package of de `dso-toolkit` package moeten onderbrengen. Wat vind jij?

De styling uit de `dso-toolkit` package willen we juist verplaatsen naar andere packages, waarom zouden we juist deze styling dan in `dso-toolkit` willen onderbrengen?

> Hoewel, headings.scss staat ook in deze map. Dat is net zo'n vreemde eend in de bijt
![image](https://user-images.githubusercontent.com/9696261/137733550-d0643f7d-11c2-46c8-a6d1-e0624126be4e.png)